jupyterlab_gitsync/cloudbuild-release.yaml

```diff
@@ -0,0 +1,33 @@
+substitutions:
```

Comment:
I know this came from the cookiecutter template but feel free to drop this as it is no longer necessary for releasing in our current repo structure.

Reply:
Okay good to know! I'll remove this file in a future PR.